torchtitan/distributed/utils.py

```diff
@@ -145,7 +145,9 @@ def set_determinism(
     # and choose a unique seed for each rank on the PP mesh.
     # We support multiple distinct dimensions by adding each distinct dimension's local rank to the seed.
     distinct_dims_in_mesh = [
```

Comment:
Thanks for catching this! I have a n00b question, will `world_mesh.mesh_dim_names` be empty or empty list: https://github.com/pytorch/torchtitan/blob/main/torchtitan/distributed/parallel_dims.py#L159, if we init_device_mesh with `mesh = init_device_mesh(device_type, dims=[], mesh_dim_names=[])`